doc/1/getting-started/kotlin/index.md

```diff
@@ -28,6 +28,32 @@ Having trouble? Get in touch with us on [Discord](http://join.discord.kuzzle.io)
 
 You can find the SDK JARs directly on [bintray](https://bintray.com/kuzzle/maven/sdk-jvm). Download and add them to your classpath.
 
+::: warning
+If you are using the `sdk-jvm-X.Y.Z.jar` in a Kotlin Android Studio project, consider adding the following lines to your `build.gradle` file:
+
+```groovy
+configurations {
+     cleanedAnnotations
+     compile.exclude group: 'org.jetbrains'
+     compile.exclude group: 'org.jetbrains.kotlin'
+ }
+```
+
+If have duplicate dependencies issues when you build your project, remove the lines above if you added them, and consider using the `sdk-jvm-X.Y.Z-without-dependencies.jar`. Then, you have to add the following dependencies:
+
+```groovy
+    implementation("io.ktor:ktor-client-websockets:1.5.2")
+    implementation("io.ktor:ktor-client-okhttp:1.5.2")
+    implementation("io.ktor:ktor-client-cio:1.5.2")
+    implementation("io.ktor:ktor-client-json:1.5.2")
+    implementation("io.ktor:ktor-client-gson:1.5.2")
+    implementation("io.ktor:ktor-client-serialization:1.5.2")
+    implementation("com.google.code.gson:gson:2.8.5")
+```
+
+:::
```

Comment:
(open to debate) I find this piece of documentation confusing: you instruct to add lines, but then to remove them if problems arise.

What about introducing these two ways of building the SDK jars in the following manner?

```
This SDK has 2 jar files that you can use:
* `sdk-jvm-<version>-without-dependencies.jar`: this is the thin jar version of the SDK, without any dependencies included in it
* `sdk-jvm-<version>.jar`: this is the fat jar version, containing the SDK and all its dependencies

Depending on your project, you might need one or the other version of the SDK : if you already use some of the dependencies needed by the SDK, then you need to use the thin jar version. Otherwise, you may use the fat jar one.
```

And then you can explain how to include the SDK as a follow-up, with how to modify the gradle file.

wdyt?

Reply:
I updated it :+1: